### DISCUSSION TITLE
Unraid Support

### DISCUSSION THREAD

**Discussion by the-Jamz** [coollabsio/coolify#2709] [Feature Requests]
Hey, in the spirit of self-hosting, how difficult would it be to support unraid servers? I would think as Docker is supported and coolify can already connect over ssh, this might not be too difficult? Currently the validation fails for me, if this is something I could help with implementing I'd be equally happy to take a look if someone could point me in the right direction.

**Comment by Fefe-Hern** [comment]
Bump, also interested if it's possible to run unRAID and Coolify on the same server. I'm considering running Coolify in a VM in unRAID showing up as a separate device, but if baremetal can handle both at the same time I can't complain.

**Reply by Shadowfita** [reply]
Nested dockers will probably let you run Coolify on Unraid

**Reply by HammyHavoc** [reply]
> Nested dockers will probably let you run Coolify on Unraid

Plugin would be preferred—when the array is stopped, Docker daemon is also stopped.

**Reply by ZyraphChron** [reply]
Just trying to run Coolify in a VM and connect to SSH to the unRAID baremetal shows that, while SSH can work for the connection, somehow it thinks Docker isn't installed. Just running "docker version" shows relevant info, but for some reason Coolify seems to explicitly deny OS support for unRAID.

I don't see what could be incompatible, since it's literally just using the unRAID resources outside the Coolify VM to build and install the Docker containers, but I'm not really a programmer.
<img width="1913" height="1033" alt="coolify-cmd-unraidserver" src="https://github.com/user-attachments/assets/022c5dad-a430-47d0-9615-e54462c9d9d9" />
<img width="1375" height="991" alt="docker-unraidserver" src="https://github.com/user-attachments/assets/218ad713-fc51-4a9d-97b2-cae00e01215a" />
<img width="1919" height="1035" alt="coolify-unraidserver" src="https://github.com/user-attachments/assets/b80c5ee7-2363-440e-83c4-007da55c2c7a" />

**Comment by sledgemasterrrr** [comment]
Did you ever get this setup in Unraid?
